scripts/jenkins/push_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Cosmetic preference :)
```suggestion
#!/usr/bin/env bash
```